pkg/plugins/policies/meshtcproute/api/v1alpha1/meshtcproute.go

```diff
@@ -60,11 +59,12 @@ type Rule struct {
 }
 
 type RuleConf struct {
-	BackendRefs *[]BackendRef `json:"backendRefs,omitempty"`
+	// +kubebuilder:validation:MinItems=1
```

Comment:
Good IMO, _if_ a rule is specified it has to point somewhere.

Reply:
Agree